Flutter/cartesian-charts/chart-types/error-bar-chart.md

```diff
@@ -86,37 +87,30 @@ You can customize the error bar depending on the error value by setting the valu
 
       return Scaffold(
         body: SfCartesianChart(
-      series: <ChartSeries<ChartData, int>>[
-        ErrorBarSeries<ChartData, int>(
-          dataSource: chartData,
-          xValueMapper: (ChartData data, _) => data.x,
-          yValueMapper: (ChartData data, _) => data.y,
-          width: 1.5,
-          type: ErrorBarType.standardError
-        )
-      ],
-    )
-    );
-    }
-
-    class ChartData {
-      ChartData(this.x, this.y);
-      final int x;
-      final int y;
-    }
+            series: <ChartSeries<ChartData, int>>[
+              ErrorBarSeries<ChartData, int>(
+                dataSource: chartData,
+                xValueMapper: (ChartData data, _) => data.x,
+                yValueMapper: (ChartData data, _) => data.y,
+                width: 1.5,
+                type: ErrorBarType.standardError)
+              ],
+            )
+         );
+      }
 
 {% endhighlight %}
 
 ![Error bar type](cartesian-chart-types-images/error_bar_type.png)
 
 ### Custom type
 
-For `custom` type, you can customize the error bar depending on the error value by setting the values for `horizontalPositiveErrorValue`, `horizontalNegativeErrorValue`, `verticalPositiveErrorValue` and `verticalNegativeErrorValue`.
+For [`custom`]([`ErrorBarType.custom`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ErrorBarType.html)) type, you can customize the error bar depending on the error value by setting the values for [`horizontalPositiveErrorValue`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ErrorBarSeries/horizontalPositiveErrorValue.html), [`horizontalNegativeErrorValue`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ErrorBarSeries/horizontalNegativeErrorValue.html), [`verticalPositiveErrorValue`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ErrorBarSeries/verticalPositiveErrorValue.html) and [`verticalNegativeErrorValue`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ErrorBarSeries/verticalNegativeErrorValue.html).
```

Comment:
remove the custom here and have proper hyperlink snippet

Reply:
removed